api/swagger-spec/swagger.yml

```diff
@@ -65,22 +65,6 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
-  /versions/{train}/{component}/latest:
```

Comment:
Why is this section removed? If that's not strictly related to `ref(swagger): change the version to v3 from v2` could you make it a separate PR or at least a separate commit?